eq-author/cypress/routingTests/routing_e2e_readme.md

```diff
@@ -0,0 +1,43 @@
+# Routing E2E Test
+
+This test spec file was created as part of the setup work for the routing refactor. It is designed to create a simple survey that utilizes all
```

Comment:
Do we plan on keeping these tests after the refactor?

Reply:
I wasn't planning on it as they're unlikely to be very reliable as they depend on Runner which could change without our knowledge?

Reply:
Although the team restructure may be stopping that as runner is code frozen?

Reply:
The publisher output sounds like a good idea once we have proven the refactor works.